std/file.d

```diff
@@ -5119,3 +5120,74 @@ string tempDir() @trusted
     myFile.write("hello");
     assert(myFile.readText == "hello");
 }
+
+
+version(FreeBSD) {}
```

Comment:
This should be inside of the function body and part of the already existing version branches so documentation generates on all platforms.

And there should be a static assert with a message inside this branch.

Reply:
And this is guaranteed not to trigger if we're not using the function?

Reply:
Yes, if it's never called on FreeBSD the assert will not trigger.

Reply:
Hi, It seems the tester triggers the assert.
Can you please help me figure out why?

Thanks